runtime/runtime_darwin.go

```diff
@@ -26,8 +26,6 @@ func (user *OSUser) CreateNew(okIfExists bool) (err error) {
 		fullname="${0}"
 		password="${1}"
 		echo "Creating user '${username}' with home directory '${homedir}' and password '${password}'..."
-		maxid=$(dscl . -list '/Users' 'UniqueID' | awk '{print $2}' | sort -ug | tail -1)
```

Comment:
It looks like this prevents us from creating a user with an id that is already taken. Is this no longer necessary?

Reply:
We used to calculate what the `uid` would be, since we knew it was the existing maximum `uid` + 1, and we would output it in the script. Then the calling code would parse the output and store the `uid`. We no longer need the `uid` so at some point we removed the code that output it and then parsed it, but I had not noticed that the code to predict the `uid` was still in there, so this was just a cleanup to remove that code. `uid` and `maxid` are never referenced anywhere else, so it is safe to remove these lines.